--- a/cmd/alps/main.go
+++ b/cmd/alps/main.go
@@ -31,7 +31,7 @@ func main() {
 		loginKey string
 		options  alps.Options
 	)
-	flag.StringVar(&options.Theme, "theme", "", "default theme")
+	flag.StringVar(&options.Theme, "theme", "alps", "default theme")
 	flag.StringVar(&addr, "addr", ":1323", "listening address")
 	flag.BoolVar(&options.Debug, "debug", false, "enable debug logs")
 	flag.StringVar(&loginKey, "login-key", "", "Fernet key for login persistence")
