--- a/config/defaults.inc.php
+++ b/config/defaults.inc.php
@@ -33,7 +33,7 @@
 //       for Mysql: key, cipher, cert, capath, ca, verify_server_cert,
 //       for Postgres: application_name, sslmode, sslcert, sslkey, sslrootcert, sslcrl, sslcompression, service.
 //       e.g. 'mysql://roundcube:@localhost/roundcubemail?verify_server_cert=false'
-$config['db_dsnw'] = 'mysql://roundcube:@localhost/roundcubemail';
+$config['db_dsnw'] = 'sqlite:////var/lib/roundcubemail/roundcube.db?mode=0600';
 
 // Database DSN for read-only operations (if empty write database will be used)
 // useful for database replication
@@ -72,7 +72,7 @@
 // ----------------------------------
 
 // log driver:  'syslog', 'stdout' or 'file'.
-$config['log_driver'] = 'file';
+$config['log_driver'] = 'stdout';
 
 // date format for log entries
 // (read https://php.net/manual/en/function.date.php for all format characters)
@@ -542,11 +542,11 @@
 // use this folder to store log files
 // must be writeable for the user who runs PHP process (Apache user if mod_php is being used)
 // This is used by the 'file' log driver.
-$config['log_dir'] = RCUBE_INSTALL_PATH . 'logs/';
+$config['log_dir'] = '/var/log/roundcubemail/';
 
 // Location of temporary saved files such as attachments and cache files
 // must be writeable for the user who runs PHP process (Apache user if mod_php is being used)
-$config['temp_dir'] = RCUBE_INSTALL_PATH . 'temp/';
+$config['temp_dir'] = '/var/lib/roundcubemail/temp/';
 
 // expire files in temp_dir after 48 hours
 // possible units: s, m, h, d, w
@@ -780,7 +780,7 @@
 // This is used to derive mime-types from the filename extension or vice versa.
 // Such a file is usually part of the apache webserver. If you don't find a file named mime.types on your system,
 // download it from https://svn.apache.org/repos/asf/httpd/httpd/trunk/docs/conf/mime.types
-$config['mime_types'] = null;
+$config['mime_types'] = '/etc/mime.types';
 
 // path to imagemagick identify binary (if not set we'll use Imagick or GD extensions)
 $config['im_identify_path'] = null;
--- a/program/include/iniset.php
+++ b/program/include/iniset.php
@@ -40,7 +40,7 @@
 }
 
 define('RCUBE_INSTALL_PATH', INSTALL_PATH);
-define('RCUBE_CONFIG_DIR',  RCMAIL_CONFIG_DIR.'/');
+define('RCUBE_CONFIG_DIR',  '/etc/roundcubemail/');
 
 // Show basic error message on fatal PHP error
 register_shutdown_function('rcmail_error_handler');
--- a/program/lib/Roundcube/bootstrap.php
+++ b/program/lib/Roundcube/bootstrap.php
@@ -70,7 +70,7 @@
 }
 
 if (!defined('RCUBE_CONFIG_DIR')) {
-    define('RCUBE_CONFIG_DIR', RCUBE_INSTALL_PATH . 'config/');
+    define('RCUBE_CONFIG_DIR', '/etc/roundcubemail/');
 }
 
 if (!defined('RCUBE_PLUGINS_DIR')) {
